zh_cn/about/website/index.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+title: "关于本站"
+lang: zh_cn
+---
+
+网站是由 Ruby 编写的 [Jekyll][jekyll] 生成的，<br>源码托管在 
+[GitHub][github-repo] 上。
+
+视觉设计是由 [Jason Zimdars][jzimdars] 完成的。<br>
+基于之前的 Ruby 视觉识别组的设计。
```

Comment:
I think can just use English, but if you want to translate this can refer to its name:

```
Ruby 视觉识别组（Ruby Visual Identity Team）
```

/cc @stomar